src/App.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {
+  Switch,
+  Route,
+  HashRouter as Router,
+} from 'react-router-dom';
+import './style/app.css';
+import Home from './pages/Home';
+import Header from './components/Header';
+import NotMatch from './pages/NotMatch';
+import SportsList from './pages/SporstList';
+
+const App = () => (
```

Comment:
##### **Testing**

- [ ] Your App should have unit tests for several components of the app and some pure functions in Redux code.
- [ ] Also it should have integration tests using React Testing Library, you can perform a mocking data fetching and Redux Store connection.
Testing is a good code practice and it helps you to make edits to your code while making sure that nothing else breaks